Modules/gcmodule.c

```diff
@@ -1062,20 +1062,38 @@ delete_garbage(PyThreadState *tstate, GCState *gcstate,
     }
 }
 
+static void
+clear_freelists(PyFreeListState *state)
+{
```

Comment:
The following functions will be moved here in the end.
* _PyTuple_ClearFreeList
* _PyFloat_ClearFreeList
* _PyDict_ClearFreeList
* _PyAsyncGen_ClearFreeLists
* _PyContext_ClearFreeList